src/app/settings/WishListSettings.tsx

```diff
@@ -146,6 +159,7 @@ class WishListSettings extends React.Component<Props, State> {
       if (reader.result && typeof reader.result === 'string') {
         const wishListAndInfo = toWishList(reader.result);
         this.props.dispatch(transformAndStoreWishList(wishListAndInfo) as any);
+        this.props.dispatch(markWishListsFetched(undefined) as any);
```

Comment:
Fold this action's result into `transformAndStoreWishList`'s reducer rather than dispatching a second action.